Full LDAP configuration & allow http

matthgyver opened the pull request:
Hi,

There is my finished version. I added this :
* LDAP configuration 
* LDAP groups to Graylog role mapping
* allowing http _(disabled by default. It's useful in testing environment)_

Hope this will be integrated ;-)

Commit:
adding ldap configuration

matthgyver committed:
Merge pull request #1 from matthgyver/ldap

adding ldap configuration

Commit:
adding allow_http param

Commit:
adding allow_http param

Commit:
adding allow_http param

Commit:
adding allow_http param

Commit:
adding allow_http param

Commit:
adding allow_http param

matthgyver committed:
Merge pull request #2 from matthgyver/allow_http

Allow http

Commit:
init draft

Commit:
correct "action" param default choice

matthgyver committed:
Merge pull request #3 from matthgyver/ldap_group

Ldap group

Commit:
finish graylog ldap group module

Commit:
update main readme

Commit:
upadate main.yml

matthgyver committed:
Merge pull request #4 from matthgyver/ldap_and_allow_http

Ldap and allow http

Commit:
minor update

matthgyver committed:
Merge pull request #5 from matthgyver/ldap_and_allow_http

minor update

tkuronen commented:
Hi,
Great addition.
This allow_http parameter would also solve a problem in production environments where SSL is terminated in SSO proxy and Graylog itself is listening only http. However Ansible is run directly to Graylog nodes bypassing the SSO.

Commit:
add input type

Commit:
add title parameter

Commit:
add global_input parameter

Commit:
add node parameter

Commit:
update gitignore

Commit:
add allow_override_date parameter

Commit:
add bind_address parameter

Commit:
add expand_structured_data parameter

Commit:
minor fix and add force_rdns parameter

Commit:
add number_worker_threads parameter

Commit:
add override_source param

Commit:
add port parameter

Commit:
add recv_buffer_size param

Commit:
add tcp_keepalive param

Commit:
add tls_enable parameter

Commit:
add tls_cert_file param

Commit:
add tls_key_file param

Commit:
add tls_key_password param

Commit:
add tls_client_auth param

Commit:
add tls_client_auth_cert_file param

Commit:
add use_null_delimiter param

Commit:
split input module by type

Commit:
create rsyslog input ok

Commit:
add update rsyslog input

Commit:
add remove input option

Commit:
add examples

Commit:
update documentation

Commit:
GELF input configration OK

Commit:
add example

Commit:
update README

matthgyver committed:
Merge pull request #6 from matthgyver/manage_inputs

Manage inputs

matthgyver committed:
Manage inputs (#7)

* update readme

* documentation fix

* renamme module rsyslog file

* rename graylog input file

* minor fix on example

matthgyver commented:
Hi,

I just added modules for input managment :
* List all inputs
* Delete input
* Add / update RSyslog input type
* Add / update GELF input type

Commit:
fix validate_certs mistake

Commit:
add no log to tls_key_password

Commit:
add option

Not create the input if one of the same name exists
Use force param to bypass this restriction

Commit:
add option

Add option to not create the input if one of the same name (title in Graylog langage) exists
Bypass this restriction with force param

Commit:
minor fix on update action
